server/src/test/kotlin/com/kamelia/hedera/rest/UserTest.kt

```diff
@@ -319,6 +319,47 @@ class UserTest {
         }
     }
 
+    @DisplayName("Updating own state (`enabled`)")
+    @ParameterizedTest(name = "as {0} to {1} is 403 Forbidden")
```

Comment:
```suggestion
    @ParameterizedTest(name = "Updating own state as {0} to {1} is 403 Forbidden")
```

Reply:
I followed some of the other tests which do the same thing (display name + param names = sentence). I don't like the repetition.